### ISSUE TITLE
Play with mpv directory option

### ISSUE THREAD

**Issue by wadixx** [Fredolx/open-tv#36]
Can you add an option to play channels with own mpv directory? This way it can take advantage of installed scripts. Also it is a lot easier to use parameters with mpv.conf file.

**Comment by Fredolx** [comment]
Isn't this doable with the global mpv params in settings? I don't use scripts so please give me more details about this and I'll definitely consider it

**Comment by wadixx** [comment]
Yes you can set parameters in settings but you can't use scripts via parameters. Scripts need to be loaded in their own folder in mpv directory. So when mpv starts it loads them automatically. Open-tv uses its own mpv so it doesn't call scripts.

**Comment by Fredolx** [comment]
Are you on Windows? If so, is mpv available on your path (you can type mpv in a powershell window and it launches mpv)
